### ISSUE TITLE
Unsatisfied linker error v1.3

### ISSUE THREAD

**Issue by Luonic** [bytedeco/javacv#580]
Hello! I have tried installing newest version of javacv, but I am getting error, while with 1.2.7 everything worked ok

Currently libs are installed this way and i copied *.so files from platform-specific jars to jniLibs: 
```
dependencies {
    compile fileTree(include: ['*.jar'], dir: 'libs')
    testCompile 'junit:junit:4.12'
    compile 'com.android.support:appcompat-v7:24.2.1'
    compile 'com.android.support:design:24.2.1'
    compile 'com.github.channguyen:rsv:1.0.1'
    compile 'org.bytedeco:javacv:+'
    compile 'org.bytedeco:javacpp:+'
    compile 'org.bytedeco.javacpp-presets:opencv:3.1.0-1.3:android-arm'
    compile 'org.bytedeco.javacpp-presets:ffmpeg:3.2.1-1.3:android-arm'
    compile 'org.bytedeco.javacpp-presets:opencv:3.1.0-1.3:android-x86'
    compile 'org.bytedeco.javacpp-presets:ffmpeg:3.2.1-1.3:android-x86'
}

configurations {
    all*.exclude group: 'org.bytedeco', module: 'javacpp-presets'
}
```

I have tried to manually install libs but getting same error. What can be a problem?

Here is classic error:
```
12-11 16:07:32.140 21559-21559/com.sleepycatstudios.slowy.jnidebug A/libc: Fatal signal 11 (SIGSEGV), code 1, fault addr 0x0 in tid 21559 (.slowy.jnidebug)
12-11 16:07:32.242 199-199/? A/DEBUG: *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
12-11 16:07:32.242 199-199/? A/DEBUG: Build fingerprint: 'google/hammerhead/hammerhead:6.0.1/M4B30X/3237893:user/release-keys'
12-11 16:07:32.242 199-199/? A/DEBUG: Revision: '11'
12-11 16:07:32.242 199-199/? A/DEBUG: ABI: 'arm'
12-11 16:07:32.242 199-199/? A/DEBUG: pid: 21559, tid: 21559, name: .slowy.jnidebug  >>> com.sleepycatstudios.slowy.jnidebug <<<
12-11 16:07:32.242 199-199/? A/DEBUG: signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86714): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86715): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86716): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86717): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86718): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86719): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86720): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86721): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86722): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86723): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86724): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86725): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86726): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86727): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86728): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86729): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86730): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86731): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86732): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86733): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86734): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86735): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86736): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86737): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86738): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86739): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86740): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86741): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86742): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.242 199-199/? W/debuggerd: type=1400 audit(0.0:86743): avc: denied { search } for name="com.sleepycatstudios.slowy.jnidebug" dev="mmcblk0p28" ino=154767 scontext=u:r:debuggerd:s0 tcontext=u:object_r:app_data_file:s0:c512,c768 tclass=dir permissive=0
12-11 16:07:32.261 199-199/? A/DEBUG:     r0 00000000  r1 9ad5a12c  r2 00000000  r3 00000072
12-11 16:07:32.261 199-199/? A/DEBUG:     r4 9adc0654  r5 00000000  r6 00000000  r7 00000000
12-11 16:07:32.261 199-199/? A/DEBUG:     r8 bed3ed78  r9 9ad3fd54  sl 00000000  fp 00000000
12-11 16:07:32.261 199-199/? A/DEBUG:     ip 9adb7a64  sp bed3ed54  lr 9abcc46c  pc 9b0f9340  cpsr 20070010
12-11 16:07:32.265 199-199/? A/DEBUG: backtrace:
12-11 16:07:32.265 199-199/? A/DEBUG:     #00 pc 0000d340  /data/app/com.sleepycatstudios.slowy.jnidebug-1/lib/arm/libavutil.so (av_strstart+16)
12-11 16:07:32.265 199-199/? A/DEBUG:     #01 pc 00168468  /data/app/com.sleepycatstudios.slowy.jnidebug-1/lib/arm/libavformat.so
12-11 16:07:32.265 199-199/? A/DEBUG:     #02 pc 0007af10  /data/app/com.sleepycatstudios.slowy.jnidebug-1/lib/arm/libavformat.so (av_probe_input_format3+180)
12-11 16:07:32.266 199-199/? A/DEBUG:     #03 pc 0007b120  /data/app/com.sleepycatstudios.slowy.jnidebug-1/lib/arm/libavformat.so (av_probe_input_format2+16)
12-11 16:07:32.266 199-199/? A/DEBUG:     #04 pc 001a16e8  /data/app/com.sleepycatstudios.slowy.jnidebug-1/lib/arm/libavformat.so (avformat_open_input+1492)
12-11 16:07:32.266 199-199/? A/DEBUG:     #05 pc 0005e880  /data/app/com.sleepycatstudios.slowy.jnidebug-1/lib/arm/libjniavformat.so (Java_org_bytedeco_javacpp_avformat_avformat_1open_1input__Lorg_bytedeco_javacpp_avformat_00024AVFormatContext_2Ljava_lang_String_2Lorg_bytedeco_javacpp_avformat_00024AVInputFormat_2Lorg_bytedeco_javacpp_avutil_00024AVDictionary_2+272)
12-11 16:07:32.266 199-199/? A/DEBUG:     #06 pc 0006e017  /data/data/com.sleepycatstudios.slowy.jnidebug/cache/slice-ffmpeg-3.2.1-1.3_6d114b05284e4a9e8dcc660a33502f11ac88ed59-classes.dex (offset 0x50000)
12-11 16:07:32.661 199-199/? A/DEBUG: Tombstone written to: /data/tombstones/tombstone_01
12-11 16:07:32.661 199-199/? E/DEBUG: AM write failed: Broken pipe
```

**Comment by saudet** [comment]
So it's probably a bug in newer versions of FFmpeg. Could you please file a ticket upstream?

**Comment by Luonic** [comment]
@saudet i will, if you direct me, what shoud i write and where, thanks. And so, ffmpeg will work again only with new release of javacv and if ffmpeg maintainers will fix these problem, right?

**Comment by saudet** [comment]
Follow the instructions here: https://ffmpeg.org/bugreports.html

Basically you'll need to build FFmpeg from source, provide some code that exhibits that error on that version, but that works fine on older versions of FFmpeg.

**Comment by saudet** [comment]
I've just released JavaCPP and JavaCV 1.4, based on FFmpeg 3.4.1. Does the crash still happen with those versions?

**Comment by saudet** [comment]
Given the lack of feedback, I'm assuming this has been fixes with recent versions of FFmpeg, but please feel free to reopen if this isn't the case! Thanks
